requirements.txt

```diff
@@ -1,4 +1,4 @@
 torchvision>=0.3.0
 pretrainedmodels==0.7.4
 efficientnet-pytorch==0.6.3
-timm==0.3.2
+git+https://github.com/rwightman/pytorch-image-models@d8e69206be253892b2956341fea09fdebfaae4e3
```

Comment:
No release is out with genet yet, so we'll have to wait until release

Reply:
Pip does not support such requirements
Possible solution: in encoders/__init__.py use `try: except` while loading this encoder with suggestion to install latest source timm (before timm new release)

Reply:
Pip supports such requirement and `pip install -r requirements.txt` works fine. It is still a temporary solution as timm releases often and in my opinion it's easier to wait.